applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_3D2N.cpp

```diff
@@ -1080,45 +1045,31 @@ namespace Kratos
 
 
 		bounded_matrix<double,msElementSize,msElementSize> aux_matrix = ZeroMatrix(msElementSize);
-		aux_matrix = prod(TransformationMatrix, rLeftHandSideMatrix);
+		aux_matrix = prod(transformation_matrix, rLeftHandSideMatrix);

```

Comment:
take out the zero initialization 
   
      bounded_matrix<double,msElementSize,msElementSize> aux_matrix =  prod(TransformationMatrix, rLeftHandSideMatrix);